### PR TITLE
console: support tracking VOLATILE functions as mutations or queries (close #6228) (#6096)

### LINKED ISSUE #6228: Console UI for SQL volatile functions
* https://github.com/0x777/graphql-engine/blob/pg-function-improvements/rfcs/pg-function-improvements.md
* https://github.com/hasura/graphql-engine-internal/issues/502

Server PR: https://github.com/hasura/graphql-engine/pull/6160

Console PR: https://github.com/hasura/graphql-engine/pull/6096

### PR DESCRIPTION
**Pull request by beerose**
### Changelog

- [ ] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Console

---

Screenshot 2020-12-08 at 15 40 21

### CODE REVIEW

**Review by iykekings — CHANGES_REQUESTED** [review]
Looks perfect to me. Great work @beerose. Can you check the error from Typescript I pointed out?

File: console/src/components/Services/Common/Notification.tsx [hunk 1]
```diff
@@ -42,11 +40,11 @@ export const showNotification = (
     dispatch(
       displayNotification(
         {
-          position: options.position || 'tr',
+          position: options?.position || 'tr',
           autoDismiss: ['error', 'warning'].includes(level) ? 0 : 5,
           dismissible: ['error', 'warning'].includes(level)
```
[the remark this answers is not in the document]
Comment (beerose) [reply console/src/components/Services/Common/Notification.tsx:45]: Fixed that recently in DT (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47560)
Will update deps

[the remark this answers is not in the document]
Comment (beerose) [reply console/src/components/Services/Common/Notification.tsx:45]: I'm bringing back the comment as `react-notification-system-redux` library needs to be updated to reflect types change.

**Review by iykekings — APPROVED** [review]

**Review by tirumaraiselvan — APPROVED** [review]
CL

**Comment by beerose** [comment]
Closing in favour of https://github.com/hasura/graphql-engine-mono/pull/109

**Pull request closed without merging**